config/saml.yml

```diff
@@ -18,3 +19,4 @@ production:
   certificate: <%= File.read(File.expand_path(ENV["SAML_CERTIFICATE_FILE"])).dump %>
   key: <%= File.read(File.expand_path(ENV["SAML_KEY_FILE"])).dump %>
   relay: <%= ENV["SAML_RELAY"] %>
+  logout_relay: <%= ENV["SAML_LOGOUT_RELAY"] %>
```

Comment:
Make sure you notify devops to update:
https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/roles/vets-api-common/vars/main.yml